.codesandbox/tasks.json

```diff
@@ -0,0 +1,22 @@
+{
+	"setupTasks": [
+		"npm install",
+		"npm run build",
+		"node ./bin/prepare-codesandbox.js"
+	],
+	"tasks": {
+		"install-dependencies": {
+			"name": "Install Dependencies",
+			"command": "npm install"
+		},
+		"start-app": {
+			"name": "Run Dev Server",
+			"command": "npm run wp-env start",
+			"runAtStart": true,
+			"preview": {
+				"port": 8888,
```

Comment:
Might be good to also expose the testing environment as well? Or is it not possible?

Reply:
It's already exported, this is basically just for the link to disable in the Github comment.